x-pack/plugins/data_visualizer/public/application/common/components/top_values/top_values.tsx

```diff
@@ -168,7 +168,8 @@ export const TopValues: FC<Props> = ({
   return (
     <ExpandedRowPanel
       dataTestSubj={'dataVisualizerFieldDataTopValues'}
-      className={classNames('dvPanel__wrapper', compressed ? 'dvPanel--compressed' : undefined)}
+      css={css({ width: '95%', overflow: 'hidden' })}
```

Comment:
Probably not worth spending much longer on, but is there a quick way of ensuring the width decreases on the top value part of the content so that the counts and filter buttons remain visible on smaller screen widths:

<img width="772" alt="Screenshot 2024-09-17 at 09 31 55" src="https://github.com/user-attachments/assets/fad4cc6a-0df1-48ad-a394-22dc580e0357">

Reply:
Looked into this but it's a bit of a rabbit hole. Noticed:
- the flex layout seems not correct because the +/- filter buttons are not wrapped in FlexItem.
- not sure but I think the progress bar does some "magical" animated resize on its own, i'm not sure if that messes with the sizing
- tried to apply EUI's `EuiTextTruncate` on the progress label but couldn't get it to work.
I wonder if the whole thing should be a table instead of flex layout?